index.html

```diff
@@ -2,12 +2,21 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard-dynamic-subset.min.css"
```

Comment:
```suggestion
      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable-dynamic-subset.min.css"
```

이렇게 변경 부탁드릴게요! static한것보다 variable 버전으로 가변 폰트를 사용하는게 좋을것 같아요!
가변 폰트로 해야 네트워크 요청이 줄고 캐싱 효율이 높아진다는 장점이 있습니당